cmd/segment-reaper/delete_test.go

```diff
@@ -0,0 +1,81 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package main
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/gogo/protobuf/proto"
+	"github.com/stretchr/testify/require"
+
+	"storj.io/storj/pkg/pb"
+	"storj.io/storj/private/testcontext"
+	"storj.io/storj/satellite/metainfo"
+	"storj.io/storj/storage"
+	"storj.io/storj/storage/teststore"
+)
+
+func TestDeleteSegment(t *testing.T) {
+	ctx := testcontext.New(t)
+	defer ctx.Cleanup()
+
+	db := teststore.New()
+	defer ctx.Check(db.Close)
+
+	{
+		err := makeSegment(ctx, db, "path1", time.Unix(10, 0))
+		require.NoError(t, err)
+
+		dryRun := false
+		deleteError := deleteSegment(ctx, db, "path1", time.Unix(10, 0), dryRun)
+		require.NoError(t, deleteError)
+		_, err = db.Get(ctx, storage.Key("path1"))
+		require.Error(t, err) // segment is deleted
```

Comment:
Add a check that this error is of the class `storage.ErrKeyNotFound`

Reply:
done